x-pack/packages/kbn-alerting-state-types/README.md

```diff
@@ -0,0 +1,8 @@
+# @kbn/alerting-state-types
```

Comment:
This package was added so that we can use the same types used in alerting and rule_registry, for rule task state, within task manager, for migration reasons. Alerting and rule_registry now import this package to get those types, and also task_manager.

No changes were made to the types. The files are not arranged in a super-logical fashion - I left them pretty much in the same files as where they originally came from, to make it easier to diff.